app/controllers/trainees/placements_controller.rb

```diff
@@ -3,6 +3,7 @@
 module Trainees
   class PlacementsController < BaseController
     include Appliable
+    include TraineeHelper
```

Comment:
I think we can just use the policy directly in the controller rather than hiding it in a helper